### PR TITLE
videointelligence: add video_analyze program

### PR DESCRIPTION
**Pull request by broady**
Includes:
* Label detection
* Shot analysis
* Explicit content detection

Use either a local file or GCS URI.

### CODE REVIEW

**Comment by broady** [comment]
R= @remi @tswast

File: videointelligence/video_analyze/generated/gen.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+)
+
+func main() {
+	tmpl, err := ioutil.ReadFile("generated/sample-template.go")
+	if err != nil {
+		log.Fatal("ReadFile:", err)
```
Comment (odeke-em) [inline videointelligence/video_analyze/generated/gen.go]: Minor nit:
```go
log.Fatalf("ReadFile: %v", err)
```

File: videointelligence/video_analyze/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Google Cloud Video Intelligence API Go example
+
+## Authentication
+
+* Create a project with the [Google Cloud Console][cloud-console], and enable
+  the [Vision API][vision-api].
```
Comment (tswast) [inline videointelligence/video_analyze/README.md]: s/Vision API/Video Intelligence API/g

Comment (broady) [reply videointelligence/video_analyze/README.md]: Done (in #334)

**Pull request closed without merging**